src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -623,7 +623,7 @@ public static string GetFunctionsExtensionVersion(this IEnvironment environment)
 
         public static bool IsTargetBasedScalingEnabled(this IEnvironment environment)
         {
-            return string.Equals(environment.GetEnvironmentVariable(TargetBaseScalingEnabled), "1");
+            return !string.Equals(environment.GetEnvironmentVariable(TargetBaseScalingEnabled), "0");
```

Comment:
In scale controller we also accept "true" and "false" values, but looking at the other flags in this class it looks like most are 0/1 only as well. If functions host only accepts 0/1, we should explicitly note this in docs

Reply:
Its a good idea to support true / false as well for boolean flags

Reply:
Function host app settings features are usually 0/1. I do not think we want to introduce different behavior for the TBS feature:
https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Environment/EnvironmentExtensions.cs